src/azul/lambda_layer.py

```diff
@@ -178,6 +183,27 @@ def _parse_reqs(self, reqs: Path) -> Iterable[Requirement]:
 
     @cached_property
     def object_key(self):
-        sha = '.'.join(file_sha1(f)
-                       for f in (self.reqs_file, self.reqs_trans_file))
-        return f'{config.lambda_layer_key}/{sha}.zip'
+        # We include requirements.txt, requirements.trans.txt, and vendored
+        # wheels in the hash to keep the layer content-addressable. The other
+        # files don't reference content in the layer, but we include them
+        # regardless since they may affect how the layer is generated. For
+        # example, Chalice is a dev-requirement, but updating may affect how
+        # dependencies are packaged.
+        #
+        # We can force the layer to be rebuilt by changing this file. To do
+        # this predictably, update the number in the next line monotonically.
+        # 0
```

Comment:
Are old zips ever cleaned up?

Reply:
No, and this has always been the case. You make a good point though, I ask @hannes-ucsc in PL.

Reply:
https://github.com/DataBiosphere/azul/issues/3453